device-types/HW-Group/HWG-STE.yml

```diff
@@ -0,0 +1,25 @@
+---
+manufacturer: HW-GROUP
+model: HWG-STE
+slug: hw-group-hwg-ste
+u_height: 0.0
+comments: '[HWg-STE (PoE)](https://www.hw-group.com/files/download/man/version/hwg-ste-man_1-0-1_en.pdf)'
+is_full_depth: false
+console-ports:
```

Comment:
As these are sensors and therefore this device is controlling them, it's probably better to use console-server-ports